jkube-kit/config/service/src/test/java/org/eclipse/jkube/kit/config/service/ApplyServiceCrudTest.java

```diff
@@ -0,0 +1,212 @@
+package org.eclipse.jkube.kit.config.service;
```

Comment:
license missing

Reply:
Seems we have a problem with the License check too :(

Reply:
I think it might be problem with license maven plugin. It doesn't seem to be working like before since we bumped it to 4.2

Reply:
yup, it's not recursive any more despite the -N flag. I'll fix it later